kong/db/dao/plugins/go.lua

```diff
@@ -36,6 +36,47 @@ do
   end
 end
 
+do
+  local pluginserver_proc
+
+  function go.manage_pluginserver()
+    if pluginserver_proc then
```

Comment:
Since `manage_pluginserver` is executed in `init_worker` phase once per worker, I don't think we need this `pluginserver_proc ` check. Simply check to see if `ngx.worker.id()` is `0` and only spawns the timer when it is will be enough.

Reply:
`go.manage_pluginserver()` is an exported function, so I didn't want to assume it will only be called once at the appropriate time.  also, we do need to keep the module-level variable, so checking if it's already set is cheaper than checking the worker id, since it's set to `true` on non-0 workers.  Not that this is in any way performance sensitive...

Reply:
We export a lot of functions, and if it is not documented in the PDK it is assumed to not be called elsewhere. In this case, I don't see any reason anyone should or would be calling it.

Reply:
As mentioned, if for defensive programming reason to keep this check, an `assert` is more appropriate than `if`, in order to not hide incorrect usage in our code base.

Reply:
done